src/main/java/uk/gov/beis/els/api/common/RescaledInternetLabelApiForm.java

```diff
@@ -6,7 +6,15 @@
 
 public class RescaledInternetLabelApiForm extends BaseInternetLabelApiForm {
 
-  @Schema(name = "Should the arrow be in colour or black and white?", description = "Use a colour arrow if you can. You can use a black and white arrow if your material is being printed in black and white. You shouldn't use black and white arrows on the internet.")
+  @Schema(description = "Use a colour arrow if you can. You can use a black and white arrow if your material is being printed in black and white. You shouldn't use black and white arrows on the internet.")
```

Comment:
I think its worth keeping the `name` field here, but we can update the text to me more clear, e.g. `name=The colour of the arrow image`

Reply:
if we define a name, the doco thinks that is the name of the paramenter. I removed it to display the param name as it should be defined
![image](https://user-images.githubusercontent.com/98316465/153245636-926e5b48-cf7e-40b0-829b-256b38ee6929.png)
^ defining a name

Reply:
Ah sorry my bad, I see what you mean. Can we add to the first part of the description then please? E..g "The colour of the arrow image. Use a colour arrow if you can. ...etc"